oidc-ui/src/App.css

```
@@ -0,0 +1,313 @@
+@import "tailwindcss";
+
+@tailwind base;
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)